monad-rpc/src/handlers/debug.rs

```diff
@@ -268,6 +294,31 @@ pub struct MonadDebugTraceTransactionParams {
     tracer: TracerObject,
 }
 
+#[derive(Serialize, Debug, schemars::JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct MonadCallFrameLog {
```

Comment:
Missing `index` field which is the index of the log in the trace. 

`Position` is the position of the log relative to subcalls in the same trace